include/fg/defines.h

```diff
@@ -172,6 +172,10 @@ typedef enum {
     FG_MARKER_STAR         = 7                     ///< Star symbol marker
 } fg_marker_type;
 
+typedef enum {
+    FG_NUMBER_FIXED         = 0,    ///< Fixed format display of numbers
+    FG_NUMBER_SCIENTIFIC    = 1     ///< Scientific format display of numbers
+} fg_display_format;
```

Comment:
The reason i named it `fg_display_format` with `number` keyword inside ENUM names is that such ENUM can be used for any future display formats for axes which are not numbers i.e. for categorical charts such as bar chart where one axes has dates or strings. We can be more precise about this enum and probably say `fg_axis_label_format`. I don't want this name to be specific to numbers but generic towards any other data that can end up as axes labels.